docs/resources/dms_rocketmq_consumer_group.md

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# huaweicloud_dms_rocketmq_consumer_group
+
+Manages DMS RocketMQ consumer group resource within HuaweiCloud.
+
+## Example Usage
+
+```HCL
+variable "instance_id" {}
+resource "huaweicloud_dms_rocketmq_consumer_group" "test" {
+  instance_id    = var.instance_id
+  name           = "consumer_group_test"
+  enabled        = "true"
+  broadcast      = "true"
+  brokers        = ["broker-0","broker-1"]
+  retry_max_time = 3
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the rocketMQ instance.
+  Changing this parameter will create a new resource.
+
+* `broadcast` - (Required, Bool) Specifies whether to broadcast of the consumer group.
+
+* `brokers` - (Required, List) Specifies the list of associated brokers of the consumer group.
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of the consumer group.
+  Changing this parameter will create a new resource.
+

```

Comment:
do "brokers" and "name" need 'forceNew'?

Reply:
done